modules/example-apache-httpd-configuration.adoc

```diff
@@ -7,7 +7,21 @@
 
 The basic identify provider (IDP) configuration in {product-title} 4 requires
 that the IDP server respond with JSON for success and failures. You can use CGI
-scripting in Apache HTTPD to accomplish this. This section provides examples.
+scripting in Apache HTTPD to accomplish this. This section provides examples
+with an httpd server configured as running a pod in cluster.
+
+.Example
+----
+oc login -u basicauth-xxia1 -p redhat
+Login successful.
+
+oc get user
+NAME              UID                                    FULL NAME
+basicauth-xxia1   3884a251-660f-46a5-a081-e21d4923d79c   Hello basicauth-xxia1
+
+IDENTITIES
+basicauth-idp:basicauth-xxia1
+----
 
 .Example `/etc/httpd/conf.d/login.conf`
 ----
```

Comment:
> I will update the docs accordingly and include the fixes for Listen

Hi, please include the "Listen" line with port 443

Reply:
Hello,

Thank you so much for your patience and help in getting this PR updated! I wanted to make sure, are you referring to replacing the ".Example `/etc/httpd/conf.d/login.conf`" section with the entire example you provided in the bug? I have an example of this in the updated doc preview here: https://deploy-preview-43061--osdocs.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-basic-authentication-identity-provider.html#example-apache-httpd-configuration_configuring-basic-authentication-identity-provider

If this is not right, I can change it back and if you would like to set up a meeting to explain further, I am happy to do so in your time zone. :)

Reply:
In last review, I did not see the line "Listen 0.0.0.0:... https" in your PR, so I commented above. Now given you added, that is great.
But you can do some necessary modification like removing some or adding some which you think are necessary.
For examples:
you don't need add "oc login -u basicauth-xxia1 -p wrong" part, it was just my negative test;
you don't need add "# Troubleshooting" part, because the doc already has troubleshooting section;
it is not good put all oc commands under "Example /etc/httpd/conf.d/login.conf", you can separately put oc commands after the final "Example /var/www/cgi-bin/fail.cgi" instead.
and so on others, like you don't need add "cat > login.cgi ...", because the doc has "Example /var/www/cgi-bin/login.cgi" section ...
All of such comments are only text arrangement, you can double check and arrange it as long as it at least reads normal in your side first.

Reply:
Suggest do double self review, this can help find above wording issues I think :), and helps improve the review iteration efficiency.

Reply:
Done! :) I also added results from example. Changes should be visible in the doc preview. Please let me know if I should remove example results.